### PR TITLE
Fixed ring_stiffener non-dimensional issues

### PR DESCRIPTION
**Pull request by dakotaramos**
## Purpose
To fix dimensional value of ring stiffeners to non-dimensional grid values, added comments.

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [ ] Maintenance update
- [ ] Other (please describe)

## Testing
We know some of the floating test cases will fail and would like feedback on how to resolve.

## Checklist
_Put an `x` in the boxes that apply._

- [ ] I have run existing tests which pass locally with my changes
- [ ] I have added new tests or examples that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation

### CODE REVIEW

File: wisdem/glue_code/gc_WT_DataStruc.py [hunk 1]
```diff
@@ -2161,7 +2161,7 @@ def setup(self):
             ivc.add_output("ring_stiffener_web_thickness", 0.0, units="m")
             ivc.add_output("ring_stiffener_flange_width", 0.0, units="m")
             ivc.add_output("ring_stiffener_flange_thickness", 0.0, units="m")
-            ivc.add_output("ring_stiffener_spacing", 0.0, units="m")
+            ivc.add_output("ring_stiffener_spacing", 0.0)
```
Comment (gbarter) [inline wisdem/glue_code/gc_WT_DataStruc.py:2164]: Do the units have to come off the axial_stiffener_spacing a few lines below too?

Comment (gbarter) [reply wisdem/glue_code/gc_WT_DataStruc.py:2164]: Maybe nevermind.  It seems like we are keeping those in deg/rad

**Comment by gbarter** [comment]
This looks good and keeps the non-dimensional approach consistent throughout.  Thank you both!

File: wisdem/glue_code/gc_WT_DataStruc.py (hunk 1 above)
Comment (johnjasa) [reply wisdem/glue_code/gc_WT_DataStruc.py:2164]: Right, we could change it to a 0 to 1 non-dimensionalization in line with the ring stiffeners, though radians doesn't vary with diameter whereas the ring_stiffener spacing would vary with member length. Do you prefer that we change this, or is it good to keep it in deg/rad?

Comment (gbarter) [reply wisdem/glue_code/gc_WT_DataStruc.py:2164]: It's okay to keep in deg/rad.

**Review by johnjasa — APPROVED** [review]

**Review by gbarter — APPROVED** [review]

**Pull request merged**